spacewalk/certs-tools/spacewalk-certs-tools.changes

```diff
@@ -1,3 +1,7 @@
+-------------------------------------------------------------------
+Fri Oct 19 16:45:00 CEST 2020 - michael.brookhuis@suse.com
```

Comment:
Remove these two header lines: they will be added by Julio in at release time

Reply:
Done